server/src/data/listings.ts

```diff
@@ -12,6 +14,45 @@ export const getListing = async (listingId: string) => {
   return listing.data();
 };
 
+export const createListing = async (title: String, description: String, price: Number, street: String, city: String, state: String, zip: String, lat: number, lon: number, ownerId: String, imageArray) => {
+  const docRef = await firestore.addDoc(collection, {
+    title: title,
+    description: description,
+    price: price.toFixed(2),
```

Comment:
by using toFixed, this makes price a string rather than a number in firestore

Reply:
changed line 21 to `price: parseFloat(price.toFixed(2)),`